File: PHP/Events/GetEventPerformancesResult.php
<?php
namespace LTD_API;

class GetEventPerformancesResult
{
    private $eventId;
    //array of PerformanceInfo
    private $performances;
    
    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getPerformances()
    {
        return $this->performances;
    }
    public function setPerformances($performances)
    {
        $this->performances = $performances;
    }
}

File: Java/Events/GetEventPerformancesResult.java
package LTD_API;

import java.util.List;

public class GetEventPerformancesResult
{
    private int eventId;
    private List<PerformanceInfo> performances;
    
    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }

    public List<PerformanceInfo> getPerformances()
    {
        return performances;
    }
    public void setPerformances(List<PerformanceInfo> performances)
    {
        this.performances = performances;
    }
}

